plugins/src/webrtcsink/imp.rs

```diff
@@ -1749,29 +1744,28 @@ impl WebRTCSink {
         let mut state = element.imp().state.lock().unwrap();
 
         if let Some(consumer) = state.consumers.get_mut(peer_id) {
+            // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
+            // that it is not worth it below that threshold
             let fec_ratio = {
-                // Start adding some FEC when the bitrate > 2Mbps as we found experimentally
-                // that it is not worth it below that threshold
-                if bitrate <= 2_000_000 || consumer.cc_info.max_bitrate <= 2_000_000 {
+                if bitrate <= DO_FEC_THRESHOLD || consumer.cc_info.max_bitrate <= DO_FEC_THRESHOLD {
                     0f64
                 } else {
-                    (bitrate as f64 - 2_000_000.)
-                        / (consumer.cc_info.max_bitrate as f64 - 2_000_000.)
+                    (bitrate as f64 - DO_FEC_THRESHOLD as f64)
+                        / (consumer.cc_info.max_bitrate as f64 - DO_FEC_THRESHOLD as f64)
                 }
             };
 
-            let fec_percentage = fec_ratio * 50f64;
-            let encoders_bitrate = ((bitrate as f64) / (1. + (fec_percentage / 100.))) as i32;
+            let fec_percentage = (fec_ratio * 50f64) as u32;
 
             if let Some(ref rtpxsend) = consumer.rtprtxsend.as_ref() {
                 rtpxsend.set_property("stuffing-kbps", (bitrate as f64 / 1000.) as i32);
             }
 
             for encoder in consumer.encoders.iter_mut() {
-                encoder.set_bitrate(element, encoders_bitrate);
+                encoder.set_bitrate(element, bitrate as i32);
```

Comment:
This reminds me that here we do not take into account the number of encoders. I think that  `webrtcsink::max-bitrate` should be the maximal bitrate **per stream**, meaning that when we set it to the bandwidth estimator we should do `max-bitrate = webrtcsink.max_bitrate * consumer.encoders.len()` and here we should end up with:

```
encoder_bitrate = ((((bitrate as f64) / (1. + (fec_percentage / 100.)))) / consumer.encoders.len()) as i32
```

@MathieuDuponchelle do you agree?

Reply:
I tried to do that, however, at the moment, when `rtpgccbwe` is created, `session.encoders` vector is still empty. I'd appreciate, if you could advise how/if this should be solved.

Reply:
I think we need to keep a ref to the bandwith estimating element and set its `max-bitrate` each time an encoder is added.

Reply:
Thanks for suggestion. 
This was basically done. Feel free to reopen the conversation, if needed :)